src/node/BrowserFetcher.ts

```diff
@@ -419,7 +419,9 @@ export class BrowserFetcher {
    * @internal
    */
   _getFolderPath(revision: string): string {
-    return path.join(this._downloadsFolder, `${this._platform}-${revision}`);
+    return path.resolve(
```

Comment:
You can just path multiple path pieces to `path.resolve`:
```js
path.resolve(this._downloadsFolder, `${this._platform}-${revision}`);
```

See: https://nodejs.org/api/path.html#path_path_resolve_paths